ansible/roles/create-vm/tasks/main.yml

```diff
@@ -2,12 +2,12 @@
 
 - name: Check family excluded
   set_fact:
-    excluded: "{{ [vm_family, vm_collection_method] in excluded_pairs }}"
+    excluded: "{{ [vm_family, vm_collection_method] in excluded_pairs or (test_platform != 'rhel-ppc64le' and vm_collection_method == 'ebpf') }}"
   when: vm_image is not defined
 
 - name: Check image excluded
   set_fact:
-    excluded: "{{ [vm_image, vm_collection_method] in excluded_pairs }}"
+    excluded: "{{ [vm_image, vm_collection_method] in excluded_pairs or (test_platform != 'rhel-ppc64le' and vm_collection_method == 'ebpf') }}"
```

Comment:
Maybe we can use a separate task to try and simplify these conditions a bit, WDYT?
```yml
  - name: Check eBPF excluded
    set_fact:
      excluded: "{{ test_platform != 'rhel-ppc64le' and vm_collection_method == 'ebpf' }}"

  - name: Check family excluded
    set_fact:
      excluded: "{{ [vm_family, vm_collection_method] in excluded_pairs }}"
    when: vm_image is not defined and excluded == false

  - name: Check family excluded
    set_fact:
      excluded: "{{ [vm_image, vm_collection_method] in excluded_pairs }}"
    when: vm_image is defined and excluded == false
```

Reply:
What about doing one more step, and try to generate `excluded_pairs`? Something like:

```yaml
vm_families: ["rhel-ppc64le"]
vm_collection_methods: [ "core_bpf", "ebpf", "magic"]
custom_excluded_pairs: [...]
excluded_pairs: "{{ [vm_families | product(methods), custom_excluded_pairs] | flatten(1) | unique }}"
```

In this way we could still tread `excluded_pairs` as a single source of truth, but assemble it from blocks. Another advantage to me is that it moves us closer to the idea of declarative test matrix. What do you think?

Reply:
I like this idea in principle, but it might be overkill since we're on the cusp of having a single collection method. Once we're just using core-bpf we can simply have a list of families / images that are excluded